src/main/java/com/avengers/gamera/controller/AuthorityController.java

```diff
@@ -0,0 +1,34 @@
+package com.avengers.gamera.controller;
+
+import com.avengers.gamera.dto.authority.AuthorityGetDto;
+import com.avengers.gamera.entity.Authority;
+import com.avengers.gamera.service.AuthorityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("authorities")
+@RequiredArgsConstructor
+public class AuthorityController {
+    private final AuthorityService authorityService;
+
+    @GetMapping
+    public List<Authority> getAuthorityList(){
```

Comment:
response authoritySlim or authorityGet dto (if need id)  instead of whole authority entity